plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/importapp/ImportApplicationWizard.java

```diff
@@ -86,8 +86,9 @@ public boolean performFinish() {
 	private boolean importProject() {
 		try {
 			final DelegatingProgressMonitor delegatingMonitor = new DelegatingProgressMonitor();
-			IStatus jobResult = WizardUtils.runInWizard(
-					new ImportJob(model.getGitUrl(), model.getCloneDestination(), delegatingMonitor), delegatingMonitor, getContainer());
+			ImportJob importJob = new ImportJob(model.getGitUrl(), model.getCloneDestination(), delegatingMonitor)
+					.setGitRef(model.getGitRef());
```

Comment:
ok, +1 then. I agree, 5 params is too much, should have a builder, but not worth the effort here.